helm/alfresco-process-application/templates/volume-init-job.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.persistence.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: volume-init
+spec:
+  template:
+    spec:
+      restartPolicy: "OnFailure"
+      imagePullSecrets:
+        - name: aae-sohel
+      volumes:
+        - name: {{ .Release.Name }}
+          persistentVolumeClaim:
+            claimName: {{ .Release.Name }}
+      containers:
+        - name: volume-init
+          image: "{{ .Values.volumeinit.image.repository }}:{{ .Values.volumeinit.image.tag }}"
+          imagePullPolicy: {{ .Values.volumeinit.image.pullPolicy }}
+          volumeMounts:
+            - name: {{ .Release.Name }}
+              mountPath: /root/{{ .Release.Name }}/1/
```

Comment:
for the sake of this spike is fine to leave "1" 

but that should be parameterised because corresponds to the application version. 
We will need parameterisation on upgrade usage.